oap-server/server-starter/src/main/resources/meter-analyzer-config/python-runtime.yaml

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: instance(['service'], ['instance'], Layer.GENERAL)
+metricPrefix: meter
+metricsRules:
+  - name: instance_pvm_process_cpu_utilization
+    exp: instance_pvm_process_cpu_utilization
+  - name: instance_pvm_process_mem_utilization
+    exp: instance_pvm_process_mem_utilization
+  - name: instance_pvm_total_cpu_utilization
+    exp: instance_pvm_total_cpu_utilization
+  - name: instance_pvm_total_mem_utilization
+    exp: instance_pvm_total_mem_utilization
+  - name: instance_pvm_gc_g0
+    exp: instance_pvm_gc_g0
+  - name: instance_pvm_gc_g1
+    exp: instance_pvm_gc_g1
+  - name: instance_pvm_gc_g2
+    exp: instance_pvm_gc_g2
+  - name: instance_pvm_gc_time
+    exp: instance_pvm_gc_time.increase("PT1M")
```

Comment:
Considering the current implementation of `MeterService`, It's more reasonable to report `gc_time` this way.
I will create a new PR for Python Agent to adapt this modification later.

Reply:
What do you mean `more reasonable`? `increase` is usually used for a counter which keeps growing.

Reply:
> Because so far gc_time is reported by MeterService every 20s but not reported right after gc.
I'm goint to make gc_time accumulatable to get average time consumption for gc within 1 minute.
Is this okay?

This is not an issue.